components/brave_wallet/browser/solana_instruction_builder.cc

```diff
@@ -69,14 +52,8 @@ std::optional<SolanaInstruction> Transfer(const std::string& from_pubkey,
 
   // Instruction data is consisted of u32 instruction index and u64 lamport.
   std::vector<uint8_t> instruction_data;
```

Comment:
`.reserve` the expected size before hand.

Reply:
or create the vector with the number of expected elements.

Reply:
https://github.com/brave/brave-core/pull/26431/commits/6809115eb5276f7e5f97305fc0e0189ade29d515